Makefile

```diff
@@ -708,9 +737,15 @@ define fast_test_folder
 	$(foreach dir, $(wildcard $(1)/*/.), $(call fast_test_folder, $(dir)))
 endef
 
+## # Run go tests
+## make test
+##
 test: $(ALL_PROTOS) tls-certs third_party/
 	$(call test_folder,.)
 
+## # Run fast go tests
```

Comment:
Nit: This runs the same tests, a better description would be
Run go tests only with worse flake and race detection

Reply:
fixed!